Desktop_Interface/isobuffer.cpp

```diff
@@ -2,391 +2,364 @@
 #include "isodriver.h"
 #include "uartstyledecoder.h"
 
-isoBuffer::isoBuffer(QWidget *parent, int bufferLen, isoDriver *caller, unsigned char channel_value) : QWidget(parent)
+isoBuffer::isoBuffer(QWidget* parent, int bufferLen, isoDriver* caller, unsigned char channel_value)
+	: QWidget(parent)
 {
-    buffer = (short *) calloc(bufferLen*2, sizeof(short));
-    bufferEnd = bufferLen-1;
-    samplesPerSecond = (double) bufferLen/(double)21;
-    samplesPerSecond = samplesPerSecond/375*VALID_DATA_PER_375;
-    sampleRate_bit = samplesPerSecond * 8;
-    virtualParent = caller;
-    channel = channel_value;
+	buffer = (short*) calloc(bufferLen*2, sizeof(short));
+	bufferEnd = bufferLen-1;
+	samplesPerSecond = (double) bufferLen/(double)21;
+	samplesPerSecond = samplesPerSecond/375*VALID_DATA_PER_375;
+	sampleRate_bit = samplesPerSecond * 8;
```

Comment:
Many of the assignments in the constructor belong in the initialiser list.

Reply:
done